app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2942,8 +2942,12 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
         out.on_edge_drop <+ removed_edges_on_node_creation_from_port;
 
         input_add_node_way <- inputs.add_node.constant(WayOfCreatingNode::AddNodeEvent);
-        input_start_creation_way <- inputs.start_node_creation.constant(
-            WayOfCreatingNode::StartCreationEvent);
+        input_start_creation_way <- inputs.start_node_creation.map(f_!([scene]
+            // Only start node creation if nothing else has focus. This is to prevent
+            // creating nodes when we are editing texts and press enter.
+            scene.focused_instance().is_none().then_some(WayOfCreatingNode::StartCreationEvent)
+        )).unwrap();
+
```

Comment:
1. I would make it a condition on the shortcut list. This way, if someone changes the shortcut to something non-colliding, then they could remove the condition of text-not-being-focused. The conditions are bool outputs, so probably we need to add an FRP output here.
2. The code actually checks if _nothing_ is focused, but comments say about _nothing else_ being focused. Perhaps the code is right (I'm not sure how the focus inside Graph Editor will work), but then let's update the comment.